presto-main/src/main/resources/webapp/assets/query.js

```diff
@@ -0,0 +1,1087 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var Table = Reactable.Table,
+    Thead = Reactable.Thead,
+    Th = Reactable.Th,
+    Tr = Reactable.Tr,
+    Td = Reactable.Td;
+
+var TaskList = React.createClass({
+    getTasks: function (stage) {
+        return [].concat.apply(stage.tasks, stage.subStages.map(this.getTasks));
+    },
+    render: function() {
+        var tasks = this.getTasks(this.props.outputStage);
+        var renderedTasks = tasks.map(function (task) {
+            var t = task;
+            var elapsedTime = parseDuration(t.stats.elapsedTime);
+            if (elapsedTime == 0) {
+                elapsedTime = Date.now() - Date.parse(t.stats.createTime);
+            }
+            return (
+                    <Tr key={t.taskStatus.taskId}>
```

Comment:
Seems like all the other tags are lower case, so I would stick with that for "Tr" and "Td" too